.github/workflows/integration.yml

```diff
@@ -412,7 +412,84 @@ jobs:
 
       - name: Run blue-green check
         if: ${{ !contains(github.workflow_ref, 'nightly') }}
-        run: php .gitlab/bin/blue-green-check.php # TODO: Move script to .github after migration
+        run: php .github/bin/blue-green-check.php
+
+  blue-green-66-67:
+    name: "PHP blue green 6.6 -> 6.7 -> 6.6"
+    runs-on: ubuntu-24.04
+    env:
+      APP_ENV: test
+      APP_URL: http://localhost:8000
+      APP_SECRET: def00000bb5acb32b54ff8ee130270586eec0e878f7337dc7a837acc31d3ff00f93a56b595448b4b29664847dd51991b3314ff65aeeeb761a133b0ec0e070433bff08e48
+      OPENSEARCH_URL: 127.0.0.1:9200
+      BLUE_GREEN_DEPLOYMENT: 1
+      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
+      PREVIOUS_MAJOR_VERSION_BRANCH: "6.6.x"
+
+    services:
+      elasticsearch:
+        image: "opensearchproject/opensearch:1"
+        env:
+          discovery.type: single-node
+          plugins.security.disabled: "true"
+        ports:
+          - "9200:9200"
+
+    steps:
+      - name: Setup previous major version
+        uses: shopware/setup-shopware@main
+        with:
+          install: "false"
+          shopware-version: ${{ env.PREVIOUS_MAJOR_VERSION_BRANCH }}
+          shopware-repository: ${{ github.repository }}
+
+      - name: Start Webserver
+        run: symfony server:start -d
+
+      - name: Install Shopware in previous major version on test DB
+        run: composer init:testdb
+
+      - name: Checkout current major version
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Update to current major version
+        env:
+          COMPOSER_ROOT_VERSION: "6.7.9999999-dev"
+          DATABASE_URL: "mysql://root@127.0.0.1:3306/shopware_test"
+        run: |
+          rm -rf vendor-bin
+          rm -rf vendor
+          composer update -o
+          bin/console system:update:finish
+
+      - name: Checkout previous major version again
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.PREVIOUS_MAJOR_VERSION_BRANCH }}
+
+      - name: Setup Composer in previous major version again
+        run: |
+          rm -rf vendor-bin
+          composer update -o
+          bin/console system:generate-jwt-secret
+
+      - name: Refresh theme in previous major version
+        if: ${{ contains(github.workflow_ref, 'nightly') }}
```

Comment:
can we move this to the job level? Also would make sense to change it to the `nightly` input. That would allow you to directly trigger the "nightly version" of the workflow

Reply:
as discussed on Slack, there are two different things executed whether this is nightly or not. fast check on every PR, slow check on nightly